main.go

```diff
@@ -93,3 +88,44 @@ func main() {
 
 	os.Exit(0)
 }
+
+func generateCycloneDx(config config.Config, lResults *linker.Results) {
```

Comment:
Personally I would name this `auditWithIQ`, the SBOM is just kinda the way in which you do.